### PR TITLE
Exclude docker containers from container inventory

### PR DESCRIPTION
**Pull request by rashmichandrashekar**
Exclude docker containers from container inventory

### CODE REVIEW

File: source/code/plugin/DockerApiClient.rb [hunk 1]
```diff
@@ -86,7 +86,10 @@ def listContainers()
             containers = getResponse(request, true, false)
             if !containers.nil? && !containers.empty?
                 containers.each do |container|
-                    ids.push(container['Id'])
+                    labels = container['Labels']
+                    if !labels.nil? && !labels['io.kubernetes.pod.uid'].nil?
```
Comment (vishiy) [inline source/code/plugin/DockerApiClient.rb]: Can you please make this a case-insensitive comparison, just in case... Thanks.

Comment (rashmichandrashekar) [reply source/code/plugin/DockerApiClient.rb]: Sure. Fixed

Comment (vishiy) [reply source/code/plugin/DockerApiClient.rb]: Hi Rashmi - I was intending to say the label value (kubernetes.pod.uid) being case insensitive (not the property name)

Comment (rashmichandrashekar) [reply source/code/plugin/DockerApiClient.rb]: Sure.

**Comment by PashaPash** [comment]
Is that a fix for "pause-amd64" container inventory?

**Comment by rashmichandrashekar** [comment]
@Pashapash: yes it is

**Pull request merged**